server/index.ts

```diff
@@ -66,4 +66,4 @@ if (require.main === module) {
   );
 }
 
-export default app;
+export default app;
```

Comment:
this should also be reverted, and hopefully small spurious changes like these won't happen again once we get our linting sorted